pytradfri/__main__.py

```diff
@@ -122,12 +129,19 @@ def dump_devices() -> None:
     print("> devices")
     print("> homekit_id")
     if light:
+        print("> lights")
         print("> light.light_control.lights")
         print("> api(light.light_control.set_dimmer(10))")
         print("> api(light.light_control.set_dimmer(254))")
         print("> api(light.light_control.set_xy_color(30015, 26870))")
         print("> api(light.light_control.set_predefined_color('Warm Amber'))")
         print("> api(lights[1].light_control.set_dimmer(20))")
+    if socket:
+        print("> sockets")
+        print("> socket.socket_control.sockets")
+        print("> from pytradfri.const.Days import TUESDAY, THURSDAY, SATURDAY, SUNDAY")
+        print("> days = TUESDAY | THURSDAY | SATURDAY | SUNDAY")
+        print("> api(socket.socket_control.set_repeat_days(days))")
```

Comment:
This would avoid the modification to `const.py`:

```suggestion
        print("> from pytradfri.smart_task import WEEKDAYS")
        print("> days = WEEKDAYS.tue | WEEKDAYS.thur | WEEKDAYS.sat | WEEKDAYS.sun")
        print("> api(socket.socket_control.set_repeat_days(days))")
```

Reply:
Please use the existing constant in `smart_task`.